examples/user-management/expo-user-management/App.tsx

```diff
@@ -5,6 +5,7 @@ import Auth from './components/Auth'
 import Account from './components/Account'
 import { View } from 'react-native'
 import { Session } from '@supabase/supabase-js'
+import React = require('react')
```

Comment:
Shouldn't it be better to use full ESM instead of shuffle both syntaxes?

```suggestion
import React from 'react'
```